### PR TITLE
Issue #8: config file 

### PR DESCRIPTION
**Pull request by makkoncept**
- config file is added along with the changes needed for smooth working (issue #12 have also been resolved)

- Necessary edits to the readme

### CODE REVIEW

**Comment by brettvanderwerff** [comment]
@makkoncept Thanks. I'm at work and can't review this right now, but will as soon as I get home!

File: config.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import os
+
+CONFIGURATION = {
+        # enter the full path of downloaded bootstrap template as the value of directory_path key
```
Comment (brettvanderwerff) [inline config.py]: I would just add to this comment to explicitly state that the folio_example occupies the configuration value by default and the user should replace this default with their own path

File: flaskerizer.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import os
 from structure_directory import StructureDirectory
 from write_app import WriteApp
+import config
```
Comment (brettvanderwerff) [inline flaskerizer.py]: Imports usually go by alphabetical order by convention, its not a deal breaker though. I miss it sometimes

**Comment by makkoncept** [comment]
apologies for late response , there is a time zone difference

**Comment by brettvanderwerff** [comment]
@makkoncept see my comment in the code, the only thing I changed was the docstring placed within the dictionary in the config file, you get an exception unless you move the docstring to outside the dictionary. Other than that it looks great. All tests are passing I can see now thanks to our new travis ci enhancement. Don;t worry about the time issue. We're lucky to have a dedicated contributor like you. You've done a ton of great work and I wanted to give you a special thank you for that. I'm merging this.

**Pull request merged**